modules/android/pages/gs-install.adoc

```diff
@@ -143,135 +141,130 @@ dependencies {
 ----
 --
 
-Vector Search Beta-1 Extension::
-+
-[IMPORTANT]
---
-Vector Search is only available for 64-bit architectures.
-You must have Couchbase Lite installed before you can use the Vector Search Extension.
---
+Community Edition::
 +
 --
-. Include the following in the `android {}` section:
+. Set the Java Version, include the following in the `android {}` section:
 +
-[source,groovy, subs="attributes+"]
+[source,kotlin, subs="attributes+"]
 ----
-
 android {
-// Required only if your project has some Kotlin source code
-  kotlinOptions { jvmTarget = 1.8}
-
   compileOptions {
-      targetCompatibility 1.8
-      sourceCompatibility 1.8
+      sourceCompatibility JavaVersion.VERSION_1_8
+      targetCompatibility JavaVersion.VERSION_1_8
   } 
-
-  buildTypes {
-    debug {
-        packagingOptions.jniLibs.useLegacyPackaging = true
-    }
-    release {
-        packagingOptions.jniLibs.useLegacyPackaging = true
-    }
-  }
-
 //   ... other section content as required by user
 }
 
 ----
 
-. Include the following in the `repositories {}` section:
+. Ensure your `repositories` section includes `mavencentral()`
 +
-[source,groovy]
+[source,kotlin, subs="attributes+"]
 ----
+
 repositories {
-  maven { url 'https://mobile.maven.couchbase.com/maven2/dev/' }
-  google()
+
   mavenCentral()
-//   ... other section content as required by user
+
+  //   ... other section content as required by user
 }
+
 ----
 
-. Include the following in the `dependencies{}` section:
+. Add {cbl-te} to the `dependencies{}` section:
 +
 [source,kotlin, subs="attributes+"]
 ----
 dependencies {
 
-  implementation 'com.couchbase.lite:couchbase-lite-android-ee-ktx:{version-maintenance-android}'
-
-  // All standard 64-bit ARM architectures
-  implementation 'com.couchbase.lite:couchbase-lite-android-vector-search-arm64-{vs-version-maintenance}'
+  implementation "com.couchbase.lite:couchbase-lite-android-ktx:{version-maintenance-android}"
 
 //   ... other section content as required by user
 }
-----
 
-.. For running on x86_64 architectures, include the following in the `dependencies{}` section instead:
-+
-[source,kotlin, subs="attributes+"]
 ----
-dependencies {
 
-  implementation 'com.couchbase.lite:couchbase-lite-android-ee-ktx:{version-maintenance-android}'
+--
 
-  implementation 'com.couchbase.lite:couchbase-lite-android-vector-search-x86_64-{vs-version-maintenance}'
 
-//   ... other section content as required by user
-}
-----
---
+=====
 
-Community Edition::
-+
+== Kotlin - Vector Search Extension Install Instructions
+
+More detailed instructions on getting up and running with the Couchbase Lite Vector Search Extension for Android (Kotlin).
+
+NOTE: The Kotlin download also includes the Java version of Couchbase Lite for Android, along with the other dependencies.
```

Comment:
Can you actually describe exactly what the download contains? 

The Kotlin download include both the Java version or....